Gemfile

```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.2'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4', '>= 6.1.4.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 5.0'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 5.0'
+# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
+gem 'turbolinks', '~> 5'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+# Use Active Storage variant
+# gem 'image_processing', '~> 1.2'
+
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', '>= 1.4.4', require: false
+
+group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+group :development do
+  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem 'web-console', '>= 4.1.0'
+  # Display performance information such as SQL time and flame graphs for each request in your browser.
+  # Can be configured to work on production as well see: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
+  gem 'rack-mini-profiler', '~> 2.0'
+  gem 'listen', '~> 3.3'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  # Add rubocop for better rails codes
+  gem 'rubocop'
+end
+
+group :test do
```

Comment:
Another gem that is great to have in test env is [simplecov](https://github.com/simplecov-ruby/simplecov), it creates a test coverage report to you, so you know where you need to add more specs :)

Reply:
It's a great suggestion! We'll have an exercise to install it at some point in the future, so it's up to you @selvipeb :)

Reply:
Thanks @the-harry and @unijewski !
I tried simplecov and it generated many stuff inside the coverage folder. I guess at this time I will put coverage folder into gitignore first?

Reply:
Yes, exactly. Add this to .gitignore and you should be good to go. 

And if you see it's trying to cover some stuff that isn't used you can filter it on spec helper or rails helper by doing something like:

```ruby
SimpleCov.start 'rails' do
  add_filter '/bin'
  add_filter '/coverage'
  add_filter '/db'
end
```

Reply:
Thanks @the-harry !
I have it updated 😃